### ISSUE TITLE
Add code coverage uploading debug

### ISSUE THREAD

**Issue by victorb** [ipfs-inactive/dev-team-enablement#144]
In case of code coverage uploading failing, we should dump the response of the codecov API so we can troubleshoot further.

Should take care that the API doesn't respond with the access keys as well.

**Comment by vmx** [comment]
It seems to be fixed as seen on the output at https://github.com/ipfs/testing/issues/154.

**Comment by VictorBjelkholm** [comment]
@vmx no, but that issue is a illustration of the problem. Right now, the little program
we use for parsing the response, is crashing and while it's doing so, doesn't
show the actual response from codecov.

There is something else than JSON in the response from codecov, and it's that
response that we should show instead of the program exception.

**Comment by daviddias** [comment]
Still an issue?
